src/router/handlers/open.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * @swagger
+ * /org/:
+ *  get:
+ *    description: Use to request organization
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        schema:
+ *          type: object
+ *          properties:
+ *            organizationName:
+ *              type: string
+ *              example: Mount Allison
+ */
+let getOrganization = async (req, res) => {
+
+};
+
+/**
+ * @swagger
+ * /hacks/:
+ *  get:
+ *    description: Use to request all hackathons
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        schema:
+ *          type: object
+ *          properties:
+ *            hackathons:
+ *              type: string
+ *              example: []
+ */
+let getHackathons = async (req, res) => {
+
+};
+
+/**
+ * @swagger
+ * /hacks/:hid/:
+ *  get:
+ *    description: Use to request a hackathon
```

Comment:
Maybe be slightly more specific: `Use to request a hackathon overview`, which is distinct from the `hackathon details`

Reply:
Agreed.